src/Client/QueryFuzzer.cpp

```diff
@@ -903,6 +903,45 @@ void QueryFuzzer::notifyQueryFailed(ASTPtr ast)
         remove_fuzzed_table(insert->getTable());
 }
 
+ASTPtr QueryFuzzer::fuzzLiteralUnderExpressionList(ASTPtr child)
+{
+    auto * l = child->as<ASTLiteral>();
+    chassert(l);
+    auto type = l->value.getType();
+    if (type == Field::Types::Which::String && fuzz_rand() % 7 == 0)
+    {
+        String value = l->value.get<String>();
+        child = makeASTFunction(
+            "toFixedString", std::make_shared<ASTLiteral>(value), std::make_shared<ASTLiteral>(static_cast<UInt64>(value.size())));
+    }
+
+    if (fuzz_rand() % 11 == 0)
```

Comment:
Do we have another method that will remove toNullable and other functions? Take one child from function as ExpressionList. Otherwise it will always converge to have all literals converted to functions

Reply:
I don't think we do, but it should be possible to add it for the functions added here (if they are a function with one of the names and only one child, randomly replace it with the child)